src/charm.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+"""GLAuth Operator Charm."""
+
+import logging
+
+from charms.operator_libs_linux.v1 import snap
+from glauth import Glauth
+from ops.charm import CharmBase
+from ops.main import main
+from ops.model import ActiveStatus, BlockedStatus, MaintenanceStatus
+
+logger = logging.getLogger(__name__)
+
+
+class GlauthOperatorCharm(CharmBase):
```

Comment:
How about `GlauthCharm`?

```suggestion
class GlauthCharm(CharmBase):
```

Reply:
Updated as suggested!